.github/workflows/accessibility.yml

```diff
@@ -0,0 +1,139 @@
+name: Accessibility tests
+on:
+  pull_request:
+    branches: [ develop ]
+
+jobs:
+  test_a11y:
+    name: Accessibility testing with ${{ matrix.package }}
+    runs-on: ubuntu-latest
+    continue-on-error: true # currently we expect these tests to fail
+    strategy:
+      matrix:
+        python-version: ["3.11"]
+        # two options for package: "pa11y" and "lighthouse"
+        # both options use axe-core ruleset for a11y tests
+        # pa11y runs faster and produces smaller artifacts
+        # lighthouse includes performance tests and other checks for non-a11y best practices
+        # for now, just use pa11y as we focus on improving a11y specifically
+        package: ["pa11y"] 
+      fail-fast: false # don't cancel lighthouse job if pa11y fails, and vice versa
+    env:
+      AMY_DATABASE_HOST: localhost
+      AMY_DATABASE_PORT: 5432
+      AMY_DATABASE_NAME: test_amy
+      AMY_DATABASE_USER: postgres
+      AMY_DATABASE_PASSWORD: postgres  
+      AMY_INSTRUCTOR_RECRUITMENT_ENABLED: True
+
+    services:
+      postgres:
+        image: postgres
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: postgres
+          POSTGRES_DB: test_amy
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        ports:
+          - 5432:5432
+      redis:
+        image: redis
+        options: >-
+          --health-cmd "redis-cli ping"
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        ports:
+          - 6379:6379
```

Comment:
I don't think we need redis for our test suite.
If other pipelines use it, we could disable it there too.

Reply:
Ok, I removed redis from this workflow and `test.yml` and it seems to work fine.